mne/time_frequency/tests/test_multitaper.py

```diff
@@ -37,17 +40,21 @@ def test_multitaper_psd():
 
     import nitime as ni
     n_times = 1000
-    x = np.random.randn(5, n_times)
+    n_channels = 5
+    data = np.random.randn(n_channels, n_times)
```

Comment:
Can you make this use `np.random.RandomState(0).randn(...)` instead? Although it should really work for any set of random numbers, we've been trying to eliminate our use of `np.random.rand*` as much as possible.